modules/test/playwright/tests/users-admin-web/usersAndOrganizations.spec.ts

```diff
@@ -736,3 +739,156 @@ test('LPD-48741 User organizations list contains no duplicate', async ({
 		editUserPage.organizationsTable.getByText(`${parentOrganization.name}`)
 	).toHaveCount(1);
 });
+
+test(
+	'Bulk deactivate user succeed',
+	{tag: '@LPD-48841'},
+	async ({apiHelpers, page, usersAndOrganizationsPage}) => {
+		page.on('dialog', async (dialog) => await dialog.accept());
+
+		const user1 = await apiHelpers.headlessAdminUser.postUserAccount();
+		const user2 = await apiHelpers.headlessAdminUser.postUserAccount();
+		const user3 = await apiHelpers.headlessAdminUser.postUserAccount();
+		const user4 = await apiHelpers.headlessAdminUser.postUserAccount();
+		const user5 = await apiHelpers.headlessAdminUser.postUserAccount();
+
+		await usersAndOrganizationsPage.goToUsers();
+
+		const userNames: string[] = [user1.name, user2.name, user3.name];
+
+		await usersAndOrganizationsPage.deActivateUsers(userNames);
+
+		for (const userName of userNames) {
+			await expect(
+				usersAndOrganizationsPage.usersTableCell(userName)
+			).not.toBeVisible();
+		}
+
+		for (const userName of [user4.name, user5.name]) {
+			await expect(
+				usersAndOrganizationsPage.usersTableCell(userName)
+			).toBeVisible();
+		}
+
+		await usersAndOrganizationsPage.filterUsers('inactive');
+
+		for (const userName of userNames) {
+			await expect(
+				usersAndOrganizationsPage.usersTableCell(userName)
+			).toBeVisible();
+		}
+	}
+);
+
+test('LPD-48841 Can share document with site member user group user', async ({
+	apiHelpers,
+	documentLibraryPage,
+	notificationsPage,
+	page,
+	siteMembershipsPage,
+}) => {
+	const userAccount = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	userData[userAccount.alternateName] = {
+		name: userAccount.givenName,
+		password: 'test',
+		surname: userAccount.familyName,
+	};
+
+	const userAccount2 = await apiHelpers.headlessAdminUser.postUserAccount();
+
+	userData[userAccount2.alternateName] = {
+		name: userAccount2.givenName,
+		password: 'test',
+		surname: userAccount2.familyName,
+	};
+
+	const userGroup = await apiHelpers.headlessAdminUser.postUserGroup();
+
+	await apiHelpers.headlessAdminUser.assignUsersToUserGroup(userGroup.id, [
+		Number(userAccount.id),
+		Number(userAccount2.id),
+	]);
+
+	apiHelpers.data.push({
+		id: `${userGroup.id}_${userAccount.id}_${userAccount2.id}`,
+		type: 'userGroupUserAccountAssociation',
+	});
```

Comment:
can we add the add to the data directly inside the `assignUsersToUserGroup` method? Check other that already do it ... in this way we don't have to add it manually when we use it